deploy/helm/flightctl/templates/db/flightctl-db-migration-job.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $root := . -}}
-{{- $baseName := printf "flightctl-db-migration-%d" .Release.Revision -}}
+{{- $baseName := printf "flightctl-db-migration-%s-%d" (now | date "20060102-150405") .Release.Revision -}}
```

Comment:
only thing we need to consider is how this will work with ACM. Today they use a static definition by name

https://github.com/cameronmwall/multicloudhub-operator/blob/0673dc0f718aa9ca10741c5027cee84dd72155e3/hack/bundle-automation/charts-config.yaml#L39